docker/localsettings.py

```diff
@@ -103,8 +103,6 @@
         # for production this ought to be set to true on your configured couch instance
         'COUCH_HTTPS': False,
         'COUCH_SERVER_ROOT': 'couch:5984',  # 6984 for https couch
-        'COUCH_USERNAME': 'commcarehq',
```

Comment:
@sheelio I think you should just make this empty string and not change `settingshelper.py`

Reply:
Updated